scripts/azure-windows.yml

```diff
@@ -71,6 +115,37 @@ steps:
     # build the unit test executable before running tests.
     cmake --build $env:AGENT_BUILDDIRECTORY\build\tiledb  -j $env:NUMBER_OF_PROCESSORS --target tiledb_unit --config Release -- /verbosity:minimal
 
+    #vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
+    #---azurite related
+    
+    if ($env:TILEDB_AZURE -eq "ON") {
+      $useTileDBInstallNodeJS = $False #$True
+      if($useTileDBInstallNodeJS) {
+        Write-Host "azurite: using custom install nodejs"
+        #Has been observed to get through testing 'successfully' 
+        #TBD: is red highlighting normal for various items in CI logs?
+        #in CI environ, doesn't work with 'local' install, works when -g'lobal' install used in the .ps1
+        & "$env:BUILD_SOURCESDIRECTORY\scripts\install-run-azurite.ps1"
+      }
+      else { #using the node/npm already present in vm
+        Write-Host "azurite: using vm install nodejs"
+        #have gotten through testing with this as install path.
+        #this code path avoids overhead of download/expand/install of alternate nodejs/azurite.
+        npm install -g azurite
+        $azuriteDataPath = (Join-Path $env:TEMP "azuriteData")
+        New-Item -ItemType Directory -Path $azuriteDataPath
+        $azuriteDebugLog = (Join-Path $env:TEMP "azuriteDebugLog.Log")
+        cmd /c "start `"azurite`" azurite-blob --silent --location $azuriteDataPath --debug $azuriteDebugLog --blobPort 10000 --blobHost 127.0.0.1"
+      }
+      tasklist | findstr /i node.exe
+    }
+    else {
+      Write-Host "TILEDB_AZURE *NOT* on"
```

Comment:
Is this necessary?

Reply:
Useful when certain multiple items could be configured in same matrix element, tho not necessary.
Less useful having a another matrix element as requested elsewhere to separate those multiple configuration items.
Removing.